app/assets/javascripts/discourse/controllers/badges/show.js.es6

```diff
@@ -25,7 +25,7 @@ export default Ember.Controller.extend(BadgeSelectController, {
 
   @computed("username", "model.grant_count", "userBadges.grant_count")
   grantCount(username, modelCount, userCount) {
-    return username ? userCount : modelCount;
+    return username ? userCount : modelCount - 1;
```

Comment:
hmm how do we know if the user hass been granted that badge or not?

Reply:
I added a new computed variable `othersCount` to exclude the current user. It will be shown only if `canShowOthers` is true.